controllers/user.js

```diff
@@ -1,9 +1,11 @@
-const bluebird = require('bluebird');
-const crypto = bluebird.promisifyAll(require('crypto'));
+const { promisify } = require('util');
+const crypto = require('crypto');
 const nodemailer = require('nodemailer');
 const passport = require('passport');
 const User = require('../models/User');
 
+const randomBytesAsync = promisify(crypto.randomBytes);
```

Comment:
Can we have the same compact one line definition as before?  Why or why not?

Reply:
This is not possible according to the [documentation](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original).